src/components/MainScreen.js

```diff
@@ -15,6 +16,19 @@ const Flexbox = styled.main`
 `;
 
 function MainScreen() {
+  const [locations, setLocation] = React.useState([]);
+
+  React.useEffect(() => {
+    Axios.get("http://localhost:3001/locations")
+      .then(response => {
+        console.log(response);
+        setLocation(response.data);
+      })
+      .catch(error => {
+        console.log(error);
+      });
```

Comment:
this could be a helper function imported from a file that would be located in the `/api` directory